### ISSUE TITLE
Missing information about how to install coffea for notebook example

### ISSUE THREAD

**Issue by kratsg** [scikit-hep/coffea#1089]
Currently, if you just run `pip install coffea` and then run a notebook example like: https://coffeateam.github.io/coffea/notebooks/processor.html -- it will fail due to missing imports, likely from a feature/contrib that needs to be put as part of the install.

I would suggest putting an example install command at the top of each notebook to indicate the requirements for running that nb?

**Comment by ikrommyd** [comment]
Is the only issue `dask.distributed` missing? If that's the case `pip install coffea[dask]` is what is required. Perhaps advertising that is the solution.

**Comment by kratsg** [comment]
> Is the only issue `dask.distributed` missing? If that's the case `pip install coffea[dask]` is what is required. Perhaps advertising that is the solution.

Yeah, I think this is going to apply to all notebooks, as it's not necessarily clear otherwise.

**Comment by ikrommyd** [comment]
Should the dask optional dependencies become part of the required dependencies since coffea is now technically dask-only @lgray?

**Comment by nsmith-** [comment]
Now the dask dependency is required, and probably we'd like to relax that.
So I think this can be closed, as https://coffea-hep.readthedocs.io/en/latest/notebooks/processing.html should run fine with just `pip install coffea` (I did not confirm this)

We could add a `!pip install dask` into the notebook source in case we eventually back out of the hard dependency on dask.
